packages/extensions/openapi-to-typespec/src/transforms/transform-arm-resources.ts

```diff
@@ -37,7 +37,8 @@ export function isGeneratedResourceObject(name: string): boolean {
 }
 
 export function replaceGeneratedResourceObject(name: string): string {
-  return generatedResourceObjects.get(name) ?? name;
+  const mappingName = generatedResourceObjects.get(name) ?? name;
+  return mappingName == "AnyObject" ? "{}" : mappingName;
```

Comment:
This is only for the renaming part of resource. I suggest to handle the `AnyObject` in the object conversion part.

Reply:
Here it is `{}` because the template only accept `Model|void`, but my intuitive tells me it is `unknown` at first. I think we should do this change as late as possible in case in other scenarios it should be `unknown` or `void`, etc.

Reply:
Let's sync offline.